src/main/ank/2.3-free-monoids.md

```diff
@@ -4,9 +4,20 @@ class Monoid m where
   mappend :: m -> m -> m
 ```
 ```kotlin:ank:silent
-interface Monoid<M> {
-  fun mempty() : M
-  fun mappend(a1: M, a2: M): M
+package arrow.typeclasses
+
+class ForMonoid private constructor() { companion object }
```

Comment:
Even though, it's directly from the arrow doc's the reader won't be familiar with either of the following type classes or the reason, why we use MonoidOf. 
Just delete the lines 9 - 11 and 
remove Semigroup and the combineAll functions.